jdaviz/core/template_mixin.py

```diff
@@ -1473,6 +1534,9 @@ def not_trace(data):
             return not is_trace(data)
 
         def is_image(data):
+            return len(data.shape) == 2
+
+        def is_cube(data):
```

Comment:
I didn't test Cubeviz so I dunno what this change means for Cubeviz. Can one still mark a 2D image loaded into Cubeviz viewer?

Reply:
The answer is yes. However, the collapsed image (2D) still outputs into "slice" column, which can be misleading. I dunno if you want to address that here or in a follow-up ticket.

Reply:
I don't think that was caused by this change, but I'll look into if I can easily just omit the slice when the mouseover layer is 2D instead of 3D in the cube viewers.

Reply:
I thought it was caused by the WCS hack we put in for Collapse product for the mouseover. 

https://github.com/spacetelescope/jdaviz/blob/54fa29de1c0efa861c40c5706511ca5d45454957/jdaviz/configs/imviz/plugins/coords_info/coords_info.py#L198

But that is not the case because I also see Slice when I mark Moment Map product, which does not use the hack anymore.

https://github.com/spacetelescope/jdaviz/blob/54fa29de1c0efa861c40c5706511ca5d45454957/jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.py#L106-L109

Reply:
fixed (just needed to check the shape of the image layer before recording the slice in the dictionary)

Reply:
You have to rebase now, there is conflict. Thanks!